### ISSUE TITLE
Missing Parameter type AWS::SSM::Parameter::Value<String>

### ISSUE THREAD

**Issue by nowiczka** [DontShaveTheYak/cloud-radar#270]
Hi! I've noticed that there is a missing parameter type applicable for an AWS type parameter: `AWS::SSM::Parameter::Value<String>` I've solved it locally by adding the next line 

"AWS::SSM::Parameter::Value<String>": r"^(\/{0,1}(?!\/))[A-Za-z0-9\/\-_]+(\.([a-zA-Z]+))?$",

to the file `cloud_radar/cf/unit/_template.py` to `validate_aws_parameter_constraints` function.

Perhaps it could be added to your package! 😊

**Comment by shadycuz** [comment]
@dhutchison you might have better context

**Comment by dhutchison** [comment]
Thanks for raising this @nowiczka!

This is the missing piece of [issue 202](https://github.com/DontShaveTheYak/cloud-radar/issues/202#issuecomment-1596292192) that I've just not had a chance to get back to. From that comment I think we can take it a little bit further than just validating the parameter type, but also tie it in to the dynamic references functionality that we already have for providing values into the rendered template. 

SSM parameters to CloudFormation templates isn't something I've used before. Would it be possible @nowiczka to provide some (appropriately redacted) examples of the parameters you supply when deploying the CloudFormation template and some sample values for the SSM parameters themselves? This will help me add in some scenarios into our unit testing of the library to ensure it functions as expected.

**Comment by nowiczka** [comment]
Sure, no problem. I've just prepared an example. Please, let me know if you need another one more elaborated. 

In the next template, I have defined two SSM parameters, `MyBucket` and `MyDatabase`. Then, I am referencing them in the `Location` and `DatabaseName`  properties of the resource  `MyTable`:
```
AWSTemplateFormatVersion: '2010-09-09'
Transform: AWS::Serverless-2016-10-31
Description:  A simple example of a template with SSM parameters.
Parameters:
  MyBucket:
    Type: 'AWS::SSM::Parameter::Value<String>'
    Description: The bucket name where all the data will be put into.
    Default: /my_parameters/bucket/name

  MyDatabase:
    Type: 'AWS::SSM::Parameter::Value<String>'
    Description: The name of the database where the table will be created.
    Default: /my_parameters/database/name

Resources:
  MyTable:
    Type: 'AWS::Glue::Table'
    Properties:
      DatabaseName: !Ref MyDatabase  # Reference the MyDatabase SSM parameter
      TableInput:
        Name: my_table
        StorageDescriptor:
          Columns:
            - Name: column_1
              Type: string
          Location: !Sub 's3://${MyBucket}/test  # Reference the MyBucket SSM parameter

```

Remember to make sure that the specified SSM parameter paths exist in your AWS Systems Manager Parameter Store with appropriate values or references before deploying this CloudFormation template.

For example, the parameter `/my_parameters/bucket/name` can store  a string `my_custum_bucket_name`
The parameter ` /my_parameters/database/name` can store a string  `my_database`
Obviously, the bucket and the database should exist in the AWS environment before deploying the template.

**Comment by dhutchison** [comment]
Thanks - looks pretty similar to how `{{resolve:ssm` functions so shouldn't be too complicated to get working in the rendering process (famous last words). Might have a bit of time to take a look at this over the weekend, but we'll see.

**Comment by dhutchison** [comment]
This was included in the v0.10.0 release so this issue can now be closed. 

@shadycuz I noticed the release notes were empty for that release but are there for the v0.11.0 release - do you know why that is?

**Comment by shadycuz** [comment]
> This was included in the v0.10.0 release so this issue can now be closed. 
> 
> @shadycuz I noticed the release notes were empty for that release but are there for the v0.11.0 release - do you know why that is?

@dhutchison it's a bug in the tool that creates the release notes. I have reported it in the past, but no one knows what causes it. I suspect it has something to do with merge conflicts or rebasing.
